.ibm/pipelines/resources/cluster_role_binding/cluster-role-binding-ocm.yaml

```diff
@@ -7,6 +7,9 @@ roleRef:
   kind: ClusterRole
   name: rhdh-k8s-plugin-ocm
 subjects:
-- kind: ServiceAccount
-  name: rhdh-k8s-plugin
-  namespace: showcase
+  - kind: ServiceAccount
+    name: rhdh-k8s-plugin
+    namespace: showcase
```

Comment:
Will this also work with the nightly namespace/job? The namespace will change to showcase-ci-nightly.

Reply:
@subhashkhileri updated, thanks for the heads up